### PR TITLE
RavenDB-13456 ability to change identity parts separator on the server-side

### PR DESCRIPTION
**Pull request by ppekrol**

### CODE REVIEW

File: src/Raven.Server/Utils/IncludeUtil.cs [hunk 1]
```diff
@@ -183,7 +183,7 @@ private static string HandlePrefixValue(object val, StringSegment prefixSegment)
         private static string ValueWithPrefix(StringSegment prefixSegment, object val)
         {
             var prefix = prefixSegment.Subsegment(0, prefixSegment.Length - 1);
-            return (prefix.Length > 0) && (prefix[prefix.Length - 1] != '/') ? null : $"{prefix}{val}";
+            return (prefix.Length > 0) && (prefix[prefix.Length - 1] != '/') ? null : $"{prefix}{val}"; // TODO [ppekrol]
```
Comment (ppekrol) [inline src/Raven.Server/Utils/IncludeUtil.cs]: Still need to look into this

File: src/Raven.Server/Documents/Handlers/HiLoHandler.cs [hunk 2]
```diff
@@ -53,7 +53,7 @@ public async Task GetNextHiLo()
                 var tag = GetQueryStringValueAndAssertIfSingleAndNotEmpty("tag");
                 var lastSize = GetLongQueryString("lastBatchSize", false) ?? 0;
                 var lastRangeAt = GetStringQueryString("lastRangeAt", false);
-                var identityPartsSeparator = GetStringQueryString("identityPartsSeparator", false) ?? "/";
+                var identityPartsSeparator = GetCharQueryString("identityPartsSeparator", false) ?? Database.IdentityPartsSeparator;
```
Comment (ppekrol) [inline src/Raven.Server/Documents/Handlers/HiLoHandler.cs:56]: Need to think if this will not cause issues when 4.2 client will talk with 5.x server

Comment (ayende) [reply src/Raven.Server/Documents/Handlers/HiLoHandler.cs:56]: How could it? it will just default to the usual one, no?

Comment (ayende) [reply src/Raven.Server/Documents/Handlers/HiLoHandler.cs:56]: Maybe a problem if you are using a multi char string, you mean? Very unlikely

**Review by ayende — APPROVED** [review]

**Pull request merged**